dynamic-layers/openembedded-layer/recipes-bsp/grub/grub-tb/CVE-2021-3981-grub-mkconfig-Restore-umask-for-the-grub.cfg.patch

```diff
@@ -0,0 +1,49 @@
+From 0adec29674561034771c13e446069b41ef41e4d4 Mon Sep 17 00:00:00 2001
```

Comment:
@kotylamichal Why is this patch required? It wasn't there before.

Reply:
@TomaszAIR It is a security patch to access grub.cfg, we can drop it from here - it is doesn't affect the GRUB building

Reply:
Let this patch stay. I just wanted to know where it came from.